README.md

```diff
@@ -1,14 +1,16 @@
 # businext.ai
 
-**Bus**iness **in**sight **ext**ractor **a**nd **i**nterpreter?
+**Bus**iness **in**sight **ext**ractor **a**nd **i**nterpreter
 
 # Configurations
 
-These configurations are to set the api keys for the various services that are used to fetch images and data from. These are to be set in a `.env` file in the root directory of the project.
+To set up this project, a Data provider, an Extraction provider and an Interpretation provider must be configured.
```

Comment:
If we ever add more providers in the future, this list will be easier to update/read

```suggestion
To set up this project, the following must be configured:

- data provider
- extraction provider
- interpretation provider

```

Also it avoids the question of whether to use an Oxford comma